app/views/layouts/_cookie_banner.html.haml

```diff
@@ -1,14 +1,18 @@
-.app-cookie-banner.app-cookie-banner--hidden{ 'aria-label': 'cookie banner', 'data-module': 'app-cookie-banner', role: 'region' }
-  .govuk-width-container
+.app-cookie-banner.app-cookie-banner--hidden.govuk-cookie-banner{'data-module': 'app-cookie-banner', "aria-label" => "Cookies on View Court Data", "data-nosnippet" => "", role: "region"}
```

Comment:
Standard: use the new json style hash syntax instead of the rocket hash syntax in all relevant locations.

Join data attributes, i.e `data: { module: 'app-cookie-banner', nosnippet: ''}`

Reply:
Done 👍